privacyidea/static/components/config/views/config.smsgateway.html

```diff
@@ -12,6 +12,12 @@
                     <a ui-sref="config.smsgateway.add" translate>
                         Create new SMS Gateway definition
                     </a></li>
+                <hr>
+                <li role="presentation">
+                    <a target="_external" ng-href="{{ 'https://privacyidea.readthedocs.io/en/v'+
+                       privacyideaVersionNumber+'/configuration/sms_gateway_config.html#'}}">
+                        <span class="glyphicon glyphicon-info-sign"></span>
+                        <span translate>Help about the SMS-Gateways</span></a></li>
```

Comment:
And again just remove the "the"

Reply:
done in b5be0d720e77db9fe1eab889e9384eb1e4c7a521.